app/src/main/java/com/zulip/android/filters/NarrowFilterToday.java

```diff
@@ -30,6 +33,10 @@ public NarrowFilterToday createFromParcel(Parcel in) {
     public NarrowFilterToday() {
     }
 
+    public NarrowFilterToday(Date date) {
```

Comment:
It would be better to rename this file as after using this constructor its the narrow would not be for today it would be for some other day!